### PR TITLE
#5464 [Manager] Refactor Cypress custom commands to speed up the tests

### PR DESCRIPTION
**Pull request by konekoya**
### Checklist

- [x] [Review Contribution Guidelines](https://oharastream.github.io/en/docs/master/contributing/)
- [x] Add at least one reviewer (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add an assignee (of course, you are the assignee by default)
- [x] Do not set milestone, projects and labels (PR labels are added automatically by Jenkins)
- [x] Link to an issue
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
- [x] Pass QA (If the failed tests are unrelated to this PR, add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/cypress/it/appBar/devTool.test.ts [hunk 1]
```diff
@@ -108,7 +108,7 @@ describe('App Bar', () => {
         });
     });
 
-    it('should able to change the topic settings', () => {
+    it('should be able to change the topic settings', () => {
```
Comment (konekoya) [inline ohara-manager/client/cypress/it/appBar/devTool.test.ts:111]: Well...

File: ohara-manager/client/cypress/support/customCommands.ts [hunk 2]
```diff
@@ -228,9 +228,11 @@ Cypress.Commands.add('createNodeIfNotExists', (nodeToCreate: NodeRequest) => {
         return cy.wrap(nodeToCreate);
       } else {
         // create if the node does not exist
-        cy.request('POST', 'api/nodes', nodeToCreate).then((createdNode) => {
-          return cy.wrap(createdNode);
-        });
+        return cy
+          .request('POST', 'api/nodes', nodeToCreate)
+          .then((createdNode) => {
+            return cy.wrap(createdNode);
```
Comment (konekoya) [inline ohara-manager/client/cypress/support/customCommands.ts:234]: TypeScript complains about not returning this...

File: ohara-manager/client/cypress/support/customCommands.ts [hunk 3]
```diff
@@ -270,32 +272,25 @@ Cypress.Commands.add(
     // Click the quickstart dialog
     cy.visit('/');
 
-    // Since we will redirect the url
-    // need to wait a little time for url applying
+    // Wait until page is loaded
     cy.wait(1000);
```
Comment (konekoya) [inline ohara-manager/client/cypress/support/customCommands.ts:276]: This wait is still necessary

File: ohara-manager/client/cypress/support/customCommands.ts [hunk 4]
```diff
@@ -328,44 +323,6 @@ Cypress.Commands.add(
   },
 );
 
-// Note: this custom command is a "heavy" command, may take almost 40 seconds to accomplish
-// make sure you have set enough timeout of defaultCommandTimeout in cypress.e2e.json
-Cypress.Commands.add(
```
Comment (konekoya) [inline ohara-manager/client/cypress/support/customCommands.ts:333]: @saivirtue I think this command is not being used in our code. So i'm removing it

Comment (saivirtue) [reply ohara-manager/client/cypress/support/customCommands.ts:333]: Indeed. :)

**Comment by konekoya** [comment]
retry manager-e2e

**Pull request merged**